README.md

```diff
@@ -70,52 +70,104 @@ Tensor is responsible for the calculation of Layer. It executes the addition, di
+You can find the source code of the core library in nntrainer/src. In order to build them, use [meson](https://mesonbuild.com/)
+```bash
+$ meson build
+The Meson build system
+Version: 0.50.1
+Source dir: /home/wook/Work/NNS/nntrainer
+Build dir: /home/wook/Work/NNS/nntrainer/build
+Build type: native build
+Project name: nntrainer
+Project version: 0.0.1
+Native C compiler: cc (gcc 7.5.0 "cc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0")
+Native C++ compiler: c++ (gcc 7.5.0 "c++ (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0")
+Build machine cpu family: x86_64
+Build machine cpu: x86_64
+...
+Build targets in project: 11
+Found ninja-1.8.2 at /usr/bin/ninja
 
-``` bash
-$ mkdir build
-$ cd build
-$ cmake  ..
-$ make
-$ sudo make install
+$ ninja -C build
+ninja: Entering directory `build'
+[41/41] Linking target test/unittest/unittest_nntrainer_internal.
+```
+
+After completion of the build, the shared library, 'libnntrainer.so' and the static library, 'libnntrainer.a' will be placed in build/nntrainer.
+```bash
+$ ls build/nntrainer -1
+d48ed23@@nntrainer@sha
+d48ed23@@nntrainer@sta
+libnntrainer.a
+libnntrainer.so
+```
+
+In order to install them with related header files to your system, use the 'install' sub-command.
+```bash
+$ ninja -C build install
+```
+Then, you will find the libnntrainer.so and related .h files in /usr/local/lib and /usr/local/include directories.
+
+By default, the command ```ninja -C build`` generates the five example application binaries (Classification, KNN, LogisticRegression, ReinforcementLearning, and Training) you could try in build/Applications. For 'Training' as an example case,
+```bash
+$ ls build/Applications/Training/jni/ -1
+e189c96@@nntrainer_training@exe
+nntrainer_training
 ```
 
-And you could test the nntrainer with Examples in Applications. There are several examples you could try and you can compile with cmake(Ubuntu) and ndk-build (Android). Tensorflow-lite is pre-requite for this example, so you have to install it by your self. If you are trying to run on the Android, it will automatically download tensorflow (1.9.0) and compile as static library.
-For the Training Examples, you can do like this:
+In order to run such example binaries, Tensorflow-lite is a prerequisite. If you are trying to run on the Android, it will automatically download tensorflow (1.9.0) and compile as static library. Otherwise, you need to install it by yourself.
 
+To run the 'Training' example, do as follows
 ```bash
-$ cd Application/Training/jni
-$ mkdir build
-$ cd build
-$ cmake  ..
-$ make
-$ ls 
-CMakeCache.txt  CMakeFiles  cmake_install.cmake  Makefile  TransferLearning
-$ ./Transfer_learning ../../res/Training.ini ../../res/
+$ pwd
+./nntrainer
+$ LD_LIBRARY_PATH=./build/nntrainer ./build/Applications/Training/jni/nntrainer_training ./Applications/Training/res/Training.ini ./Applications/Training/res/
```

Comment:
In my case, 
```bash
$ LD_PRELOAD=./build/nntrainer/libnntrainer.so ./build/Applications/Training/jni/nntrainer_training ./Applications/Training/res/Training.ini ./Applications/Training/res/
./Applications/Training/res/happy/happy1.bmp
./Applications/Training/res/happy/happy2.bmp
./Applications/Training/res/happy/happy3.bmp
./Applications/Training/res/happy/happy4.bmp
./Applications/Training/res/happy/happy5.bmp
./Applications/Training/res/sad/sad1.bmp
./Applications/Training/res/sad/sad2.bmp
./Applications/Training/res/sad/sad3.bmp
./Applications/Training/res/sad/sad4.bmp
./Applications/Training/res/sad/sad5.bmp
./Applications/Training/res/soso/soso1.bmp
./Applications/Training/res/soso/soso2.bmp
./Applications/Training/res/soso/soso3.bmp
./Applications/Training/res/soso/soso4.bmp
./Applications/Training/res/soso/soso5.bmp
Segmentation fault (core dumped)
```

Reply:
It is because it does not have data set files which generated by nntrainer. This is going to be fixed in a coming PRs.